cfme/tests/cli/test_appliance_console.py

```diff
@@ -822,7 +821,11 @@ def test_appliance_console_evm_stop():
             4.
             5. Cross-check service stopped.
     """
-    pass
+    appliance = temp_appliance_preconfig_funcscope
+    command_set = ("ap", RETURN, "16", TimedCommand("Y", 60))
+    appliance.appliance_console.run_commands(command_set, timeout=30, output=True)
+    wait_for(lambda: appliance.is_web_ui_running(), delay=10, timeout=300, fail_condition=True,
+             message='Waiting to stop EVM service')
```

Comment:
@dgaikwad I think we should extend this test a bit more. I see here:

https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/cli/test_appliance_console.py#L855

That we test turning on evmserverd via the appliance_console. But I don't see a test that is turning both on and off the evmserverd via the appliance console. Thoughts on adding that in here? Should be fairly easy

Reply:
Hey @jawatts , Thank you for comment.
There had steps in one test case(https://github.com/ManageIQ/integration_tests/pull/9814) that were doing the same which you suggested but test case name was different so we changed test case steps accordingly to testcase name.
Now here I think instead of changing the purpose of the current test case we can add a new testcase in Polarion that can perform stop and start EVM server via appliance_console separately.
What is your opinion?
till now I incorporated remaining your review comments.